src/app/Components/LanguageAvatar.svelte

```diff
@@ -0,0 +1,56 @@
+<script lang="ts">
+  export let name: string;
+  export let selectedLanguage: string = "";
+
+  export let isSelected: boolean = name === selectedLanguage;
+  export let className: string = "avatar";
+  if (isSelected) {
+    className = className + " selected";
```

Comment:
**[Question]**: I would avoid magic string as much as possible

Reply:
Would love to discuss what you've used as workarounds for magic strings for styling as I'm not too sure how to refactor this.

Would you add another variable like this?

```typescript
let selectedClassName: string = " selected";
```

And then change line 8 to:
```typescript
className = className + selectedClassName; 
```

Reply:
In Svelte I would **not** use strings like this for class names, since there's a better way! 

Remove the `className` variable entirely, then in where you need to use the markup, do the following:

```svelte
<div class="avatar" class:selected>
    ...
</div>
```

Reply:
I'll look into this, thanks for the resource!